torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -373,6 +374,11 @@ class FullyShardedDataParallel(nn.Module):
             avoid sharding specific parameters when using an
             ``auto_wrap_policy`` or if parameters' sharding is not managed by
             FSDP. (Default: ``None``)
+        device_id (Optional[int]): An `int` describing the CUDA device FSDP
+            module should be moved to before initialization such as sharding
```

Comment:
nit: A bit confused on the comment. Maybe it should be `module should be moved to before initialization when sharding takes place`?